src/styles/Home.module.css

```diff
@@ -1,18 +1,18 @@
 .listings_container {
-    padding: 1em;
-    display: grid;
+  padding: 1em;
+  display: grid;
 }
 
 @media (min-width:40rem) {
```

Comment:
Shouldn't the queries be in `px` rather than `rem` converting `rem` to `px` for width and height wouldn't be something I personally recommend